pkgs/applications/networking/onedriver/default.nix

```diff
@@ -0,0 +1,56 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, pkg-config
+, webkitgtk
+, glib
+, fuse
+, installShellFiles
+}:
+
+buildGoModule rec {
+  pname = "onedriver";
+  version = "0.13.0-2";
```

Comment:
[v0.14.0](https://github.com/jstaf/onedriver/releases/tag/v0.14.0) is out. It makes sense to package this directly.

Reply:
Version 0.14.0 has some major bugs which lead to files corruption on the OneDrive itself (see [here](https://github.com/jstaf/onedriver/issues/350)), I decided to settle for version 0.13.0-2 which is more stable, I've been using this version for a while now without issues.